x-pack/plugins/ml/server/models/data_recognizer/modules/sample_datasets_weblogs/manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+  "id": "sample_dataset_weblogs",
+  "title": "Kibana Sample Data Web Logs",
+  "description": "Find anomalies in Kibana Sample Data for Web Logs",
```

Comment:
I'd be more specific with the description. Something like `Find anomalies in average file size for the sample data`?